shared/chat/conversation/input-area/normal/mention-input.js

```diff
@@ -149,21 +175,17 @@ class MentionInput extends React.Component<MentionInputProps, MentionState> {
     this._replaceWordAtCursor(`@${u} `)
     this._setMentionPopupOpen(false)
 
-    // This happens if you type @notausername<enter>. We've essentially 'picked' nothing and really want to submit
-    // This is a little wonky cause this component doesn't directly know if the list is filtered all the way out
     if (options && options.notUser) {
-      this._forceSubmit()
+      this._maybeSubmitAfterInsert(options.notUser)
```

Comment:
`options.notUser` is always true here...you can instead do:

```
this._maybeSubmitAfterInsert(options && options.notUser)
```
or if flow complains about not being a bool
```
this._maybeSubmitAfterInsert(options ? options.notUser : false)
```